src/sagemaker/image_uris.py

```diff
@@ -119,8 +137,12 @@ def retrieve(
             "pytorch-1.6.0-gpu-py36": "cu110-ubuntu18.04",
             "pytorch-1.6-gpu-py3": "cu110-ubuntu18.04-v3",
             "pytorch-1.6.0-gpu-py3": "cu110-ubuntu18.04",
+            "pytorch-1.8.1-gpu-py3": "cu111-ubuntu18.04"
```

Comment:
This is an additional code change we are doing, do we have a corresponding unit test to ensure we pick up the correct container version for pytorch 1.8? if not, please add one

Reply:
- This was a redundancy(as it is already specified in HF and PyTorch json files. I have removed the last key value pair. 
- The test for latest PyTorch version with HF is tested by [./tests/integ/test_huggingface.py#L34](https://github.com/jeniyat/sagemaker-python-sdk/blob/master/tests/integ/test_huggingface.py#L34)